### ISSUE TITLE
Talkback does not read "prev" and "next" arrows correctly when double tapped

### ISSUE THREAD

**Issue by rezeaka** [prolificinteractive/material-calendarview#833]
**Samsung Galaxy Note 5, Talkback version 5.2.1**
The "previous" and "next" buttons are functional, but Talkback is not reading content for them that makes any sense to the user.

- [ ] If we double-tap on an the "previous" or "next" arrow, Talkback reads "Showing item two-thousand-four-hundred-three of four-thousand-eighty-one". 

- [ ] Also, the number Talkback reads (2403) is incremented to "two-thousand-four-hunder-four" if the user double tap to go forward to the next month (October). This incrementing behavior continues each time the user double taps to go forward a month, and the number decrements if the user double taps the "previous" arrow to go back a month.

**Comment by quentin41500** [comment]
@rezeaka I wasn't able to reproduce. I turned on talkback on my phone and clicked the arrows and talback said:

- Go to previous button
- Go to next button

Not sure what you are seeing.

**Comment by rezeaka** [comment]
@quentin41500 Please go one step further by **double-tapping the arrows**. When double-tapping on the previous and next arrows is when we encounter Talkback reading "Showing item two-thousand-four-hundred-three of four-thousand-eighty-one".

I have attached a video of the issue (NOTE: I had to zip it as .mov format was not allowed as an upload file type)
[IMG_9271.zip](https://github.com/prolificinteractive/material-calendarview/files/2258035/IMG_9271.zip)
